docs/BUILD.bazel

```diff
@@ -31,6 +31,14 @@ stardoc_with_diff_test(
     bzl_library_target = "//py/private:py_pex_binary",
 )
 
+stardoc_with_diff_test(
+    name = "py_image_layer",
+    bzl_library_target = "//py:defs",
```

Comment:
this should point to the py_image_layer.bzl so you get the module docstring too

Reply:
yup, just fixed it